x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/bucket_nesting_editor.tsx

```diff
@@ -42,7 +47,7 @@ export function BucketNestingEditor({
     .map(([value, c]) => ({
       value,
       text: c.label,
-      fieldName: getFieldName(fieldMap, c),
+      fieldName: getFieldName(c, getFieldByName),
```

Comment:
Why passing `getFieldName` to itself?

Reply:
The `getFieldName` is a different function which encapsulate some more logic around validation.

Reply:
Ah got it, I misread, thanks.